examples/AspNetCore/SecretStoreConfigurationProviderSample/SecretStoreConfigurationProviderSample.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'blq' " />
```

Comment:
What is this?

Reply:
I'm not sure, it looks like visual studio added something to that project file without me noticing. I'll remove it.